### ISSUE TITLE
No matching distribution found for marshmallow==3.4.0

### ISSUE THREAD

**Issue by gusingh-r7** [marshmallow-code/marshmallow#1514]
Please refer TravisCI error below and suggest what could be the issue here?. I was able to build my application locally with `marshmallow==3.4.0`
```
Collecting marshmallow==3.4.0 (from [secure]==3.4.0)
  Could not find a version that satisfies the requirement marshmallow==3.4.0 (from [secure]==3.4.0) (from versions: 0.1.0, 0.2.0, 0.2.1, 0.3.0, 0.3.1, 0.4.0, 0.4.1, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.4, 0.5.5, 0.6.0, 0.7.0, 1.0.0a0, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 2.0.0a1, 2.0.0b1, 2.0.0b2, 2.0.0b3, 2.0.0b4, 2.0.0b5, 2.0.0rc1, 2.0.0rc2, 2.0.0, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.2.0, 2.2.1, 2.3.0, 2.4.0, 2.4.1, 2.4.2, 2.5.0, 2.6.0, 2.6.1, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.8.0, 2.9.0, 2.9.1, 2.10.0, 2.10.1, 2.10.2, 2.10.3, 2.10.4, 2.10.5, 2.11.0, 2.11.1, 2.12.0, 2.12.1, 2.12.2, 2.13.0, 2.13.1, 2.13.2, 2.13.3, 2.13.4, 2.13.5, 2.13.6, 2.14.0, 2.15.0, 2.15.1, 2.15.2, 2.15.3, 2.15.4, 2.15.5, 2.15.6, 2.16.0, 2.16.1, 2.16.2, 2.16.3, 2.17.0, 2.18.0, 2.18.1, 2.19.0, 2.19.1, 2.19.2, 2.19.3, 2.19.4, 2.19.5, 2.20.0, 2.20.1, 2.20.2, 2.20.3, 2.20.4, 2.20.5, 3.0.0a1, 3.0.0b1, 3.0.0b2, 3.0.0b3, 3.0.0b4, 3.0.0b5, 3.0.0b6, 3.0.0b7, 3.0.0b8, 3.0.0b9, 3.0.0b10, 3.0.0b11, 3.0.0b12, 3.0.0b13, 3.0.0b14, 3.0.0b15, 3.0.0b16, 3.0.0b17, 3.0.0b18, 3.0.0b19, 3.0.0b20, 3.0.0rc1, 3.0.0rc2, 3.0.0rc3, 3.0.0rc4, 3.0.0rc5)
No matching distribution found for marshmallow==3.4.0 (from [secure]==3.4.0)
```

**Comment by sloria** [comment]
I am not able to reproduce this. It's possible that this was a temporary issue on PyPI's end. Can you try again?

**Comment by gusingh-r7** [comment]
I re-tried and getting the same error message. Please let me know what details I can share with you to look into this issue.

**Comment by lafrech** [comment]
Are you using alternative repositories (other than PyPI)?

**Comment by gusingh-r7** [comment]
@lafrech It seems marshmallow 3.4.0 is not supported by python2 which is causing the issue. Is there any version available which is compatible with Python2.

**Comment by deckar01** [comment]
The latest available version in the error message is "3.0.0rc5", which is when mashmallow v3 started requiring python 3.

https://github.com/marshmallow-code/marshmallow/blob/dev/CHANGELOG.rst#300rc6-2019-05-05

**Comment by lafrech** [comment]
Bottom line, if you wish to use marshmallow 3, you should move to Python 3. (You should move to Python 3 in any case, but I guess you have reasons to be stuck on Python 2.)
